src/Provider/Geonames/Geonames.php

```diff
@@ -92,6 +98,75 @@ public function reverseQuery(ReverseQuery $query): Collection
     }
 
     /**
+     * @param string|null $country
+     * @param string|null $locale
+     * @param int|null    $maxRows
+     * @param int|null    $startRow
+     *
+     * @return array
+     *
+     * @throws \Geocoder\Exception\Exception
+     */
+    public function getCountryInfo(string $country = null, string $locale = null, int $maxRows = 10, int $startRow = 0): array
```

Comment:
@Nyholm It seems Geonames is not respecting `maxRows`. Shall we drop support for it?

http://api.geonames.org/countryInfoJSON?username=sriharithalla&style=full